Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -11,19 +11,63 @@ static void Main(string[] args)
             Test("42*?7=1974", 4);
             Test("42*?47=1974", -1);
             Test("2*12?=247", -1);
-            Console.ReadKey(true);
+            // Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            string A = string.Empty;
+            string B = string.Empty;
+            string C = string.Empty;
+            int firstCount = 0;
+            int secondCount = 0;
+            var result = 0;
+            for (int i = 0; i < equation.Length; i++){
+                if(equation[i] == '*'){
+                    firstCount = i;
+                }else if(equation[i] == '='){
+                    secondCount =i;
+                }
+            }
+            A = equation.Substring(0,firstCount);
+            int length = secondCount - firstCount;
+            B = equation.Substring(firstCount+1,length-1);
+            length = equation.Length - secondCount;
+            C = equation.Substring(secondCount+1,length-1);
+            
+            if(A.Contains("?")){
+                result = Int32.Parse(C) / Int32.Parse(B);

```

Comment:
Make result variable to string here so that you don't need to convert it every time when you need to result variable it.